### PR TITLE
Support pluck method

### PR DESCRIPTION
**Pull request by ihatov08**
Support [pluck](https://apidock.com/rails/ActiveRecord/Calculations/pluck) method.

### CODE REVIEW

**Comment by syguer** [comment]
Looks good.
but I'm concern about it can be called only as ClassMethod, not as method chain. I often see that `#pluck` called as method chain like `Book.where('created_at > ?', Date.yesterday).pluck(:id)`.
Although, I know it is difficult on ActiveHash 😇

File: spec/active_hash/base_spec.rb [hunk 1]
```diff
@@ -404,6 +404,19 @@ class Region < ActiveHash::Base
...
+    before do
+      Country.data = [
+        {:id => 1, :name => "US"},
+        {:id => 2, :name => "Canada"}
+      ]
+    end
+
+    it "returns an Array of attribute values" do
+      expect(Country.pluck(:id, :name)).to match_array([[1,"US"], [2, "Canada"]])
+    end
```
Comment (syguer) [inline spec/active_hash/base_spec.rb]: if calling `#pluck` with single symbol(`Country.pluck(:id)`),  it returns flatten array. right?(not like `[[1],[2]]`)
so please add the case

File: spec/active_hash/base_spec.rb [hunk 2]
```diff
@@ -404,6 +404,23 @@ class Region < ActiveHash::Base
...
+      ]
+    end
+
+    it "returns an two dimensional Array of attributes values" do
+      expect(Country.pluck(:id, :name)).to match_array([[1,"US"], [2, "Canada"]])
+    end
+
+    it "returns an Array of attribute values" do
+      expect(Country.pluck(:id)).to match_array([1,2])
+    end
```
Comment (ihatov08) [inline spec/active_hash/base_spec.rb]: @syguer 
Add test for single symbol case.

Comment (syguer) [reply spec/active_hash/base_spec.rb]: great 👍

**Pull request merged**